mimesis/builtins/nl.py

```diff
@@ -0,0 +1,35 @@
+from mimesis.builtins.base import BaseSpecProvider
+
+
+class NetherlandsSpecProvider(BaseSpecProvider):
+    """Provides data specific to nl locale"""
+
+    class Meta:
+        name = 'nl_provider'
```

Comment:
`netherlands_provider`

Reply:
Good catch! I changed it and added an additional test to cover it properly.